jdaviz/app.vue

```diff
@@ -189,3 +190,246 @@ export default {
   }
 };
 </script>
+
+<style id="web-app">
```

Comment:
Why so many front-end style changes from here onwards in `app.vue`? Are these really related or something was funky during rebase?

Reply:
I didn't write this. @kecnry – do you know?

Reply:
This should not be here, see #2580

Reply:
I've checked and this appears to be the only part of the diff from #2580 that was rebased poorly.